inc/functions_upload.php

```diff
@@ -119,14 +112,7 @@ function remove_attachments($pid, $posthash="")
 		$query = $db->simple_select("attachments", "*", "pid='$pid'");
 	}
 
-	if(defined('IN_ADMINCP'))
-	{
-	    $uploadpath = '../'.$mybb->settings['uploadspath'];
-	}
-	else
-	{
-	    $uploadpath = $mybb->settings['uploadspath'];
-	}
+	$uploadpath = $mybb->settings['uploadspath'];
```

Comment:
So what will happen if the user uses a relative path now? (I'm talking about the ACP here.) I will test this PR.

Reply:
Any relative path is converted to an absolute path just after settings are loaded in `inc/init.php`:

https://github.com/mybb/mybb/pull/4342/files#diff-09ba99a151ee25a01bd649b01fe94fab3b52d58c19f228a73a11e7b9050978a2R204-R209